engine/src/main/java/org/terasology/engine/module/ModuleManager.java

```diff
@@ -166,33 +186,74 @@ void loadModulesFromClassPath() {
                 System.getProperty("java.class.path").split(System.getProperty("path.separator", ":"))
         ).map(Paths::get).collect(Collectors.toList());
```

Comment:
If you want to prepare this to `gestalt v7` you shouldn't to use `Path`  class.
`gestalt v7` don't use `Path` class, because it  not exists in Android.